windows/runtime.go

```diff
@@ -162,6 +162,14 @@ func (r *Runtime) Events(ctx context.Context) <-chan *containerd.Event {
 	return r.events
 }
 
+func (r *Runtime) Pause(ctx context.Context, c containerd.Container) error {
+	return fmt.Errorf("Windows containers do not support pause")
```

Comment:
Actually windows do support pause/resume if the container is running under HyperV. It's a matter of calling `Pause()`/`Resume()` on the `hcsshim.Container` type.

You can check that a container was started using HyperV by checking the `UseHyperV` flag within the windows runtime container object.